yazi-config/preset/yazi.toml

```diff
@@ -26,27 +26,27 @@ ueberzug_offset = [ 0, 0, 0, 0 ]
 
 [opener]
 edit = [
-	{ exec = '${EDITOR:=vi} "$@"', desc = "$EDITOR", block = true, for = "unix" },
+	{ exec = '${EDITOR:=vi} "$0"', desc = "$EDITOR", block = true, for = "unix" },
```

Comment:
Using `$@` instead of `$0` can better respect the user's behavior. When the user selects a single file or uses `open --hovered`, it will only open that file. When the user selects multiple files, it will open all of them.

Reply:
Yeah sure.

`$0` => Always the file the user hovered over.
`$@` => Always the files the user selected.

When the user hasn't selected any files, only hovered over a file, the hovered file is treated as the selected file.